### ISSUE TITLE
Configuration start weather forecast

### ISSUE THREAD

**Issue by GerdRuetten** [FL550/dwd_weather#61]
The weather forecast for the coming days starts with the current day. 
The information for this day does not offer any added value since it is already displayed as "current values".
It would make more sense if the forecast started with the following day and thus another day could be shown in the locvelance map.

**Comment by nikipore** [comment]
I disagree: The info is min/max temperature not current temperature and is not redundant for the current day.

**Comment by GerdRuetten** [comment]
Min/Max is an attribut from the actual day

![image](https://user-images.githubusercontent.com/97361636/209440776-77e3fbdb-776a-486a-915b-fac11924b4f7.jpeg)

**Comment by nikipore** [comment]
Depends on the weather card:

![DCE19CBB-E5AE-4446-A69E-6774F5825438](https://user-images.githubusercontent.com/640781/209441906-b3f3af5d-e3c9-4188-82d5-e343e93eca71.jpeg)

**Comment by GerdRuetten** [comment]
Compromise the start day is configurable

**Comment by FL550** [comment]
> Min/Max is an attribut from the actual day
> 
> ![image](https://user-images.githubusercontent.com/97361636/209440776-77e3fbdb-776a-486a-915b-fac11924b4f7.jpeg)

This data is derived from the forecast values. The weather entity itself has no min/max information: https://developers.home-assistant.io/docs/core/entity/weather/

So if the forecast data starts one day later, you are loosing the min/max for the current day.

If you would like to see more days in your card, you have to address your wish to the developer of the weather card.
